inception/inception-telemetry/src/main/java/de/tudarmstadt/ukp/clarin/webanno/telemetry/ui/TelemetrySettingsPage.html

```diff
@@ -25,13 +25,10 @@
           <div class="flex-h-container flex-gutter">
             <div class="flex-content">
               <p>
-                In order to support the development of <wicket:message key="product.name"/>, it would
-                like to collect some data and send it home. Please review the settings below and 
-                opt in or out of the respective data collections. Once you are done, use the
-                <b><wicket:message key="save"/></b> button on the right side to save your choices. If
-                there were any changes to the data collection process, this page will appear again
-                when an administrator logs in. You can always revise your choices by accessing this
-                page via the <i class="fa fa-heartbeat"></i> icon in the page footer.
+                <wicket:message key="settingsIntro">
```

Comment:
I think I've found a way to use a translation properties file with HTML content without escaping HTML. I created wicket elements for texts inside tags (like <b>) and translated them separately. Do you think this is the right way to do it? I'd prefer all the translations to remain centralised in the properties files because this will be less complicated to maintain translations later on.

Reply:
Actually, that is not necessary. When embedding strings in a page using `wicket:message`, then the strings in the properties files can contain markup. Only if the strings are included through a `ResourceModel`, we would need to turn off escaping to support markup. In fact, we should try to work with `wicket:message` as much as possible and use `ResourceModel` only for cases where the text in a component needs to change dynamically (i.e. where the message key is not known or not fixed).

Reply:
I made a commit to change the Telemetry page accordingly.